x/utils/logging.go

```diff
@@ -40,28 +40,25 @@ func (r Logging) Deliver(ctx weave.Context, store weave.KVStore, tx weave.Tx, ne
 	return res, err
 }
 
-// logDuration writes information about the time and result
-// to the logger
-func logDuration(ctx weave.Context, start time.Time, msg string,
-	err error, lowPrio bool) {
-
+// logDuration writes information about the time and result to the logger
+func logDuration(ctx weave.Context, start time.Time, msg string, err error, lowPrio bool) {
 	delta := time.Now().Sub(start)
-	logger := weave.GetLogger(ctx).With("duration", micros(delta))
+	logger := weave.GetLogger(ctx).With("duration", delta/time.Microsecond)
+
 	if err != nil {
 		logger = logger.With("err", err)
 	}
 
-	// now, write it
-	if err == nil && lowPrio {
-		logger.Debug(msg)
-	} else if err != nil && !lowPrio {
+	// Although message can be empty, we still want to emit a log entry
+	// because it contains other relevant information beside the message.
```

Comment:
Thanks for the comment!
An alternative and a more log search tool friendly solution would have been a default msg and the given `msg` as logger value like: `logger = logger.With("resLog", msg)`.